hs_core/views/serializers.py

```diff
@@ -81,6 +91,15 @@ class ResourceListItemSerializer(serializers.Serializer):
     science_metadata_url = serializers.URLField()
 
 
+class ResourceType(object):
+    def __init__(self, resource_type):
+        self._resource_type = resource_type
+
+    @property
+    def resource_type(self):
+        return self._resource_type
```

Comment:
can get rid of this by just dropping the prefix underscore from `_resource_type`